ui/src/EnvVarForm/components/EnvVarForm.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+import './EnvVarForm.css'
+import Form from 'react-bootstrap/Form'
+import FormGroup from 'react-bootstrap/FormGroup'
+import Button from 'react-bootstrap/Button'
+import FormControl from 'react-bootstrap/FormControl'
+import Dropdown from 'react-bootstrap/Dropdown'
+import DropBtn from '../../CustomElements/DropBtn.js'
+
+class EnvVarForm extends React.Component {
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            envvars: {},
+            items: [],
+        }
+    }
+
+    updateVars() {
+        console.log("Fetching configmap from vars")
+        fetch('/services/jsp-api/api/user/configmap', {method:'GET'})
```

Comment:
Can we make this a global? And ideally even split to prefix (`/services/jsp-api/`) and the actual endpoint (`api/user/configmap`)? And define it at one pleace?

**EDIT:** Looking at other files - it seems the same `fetch` is called all over the place - can you refactor it to a single function `get_user_configmap` and call that instead of having the same code all over the place?